src/app/player/player.module.ts

```diff
@@ -1,12 +1,14 @@
-import {NgModule} from '@angular/core';
-import {CommonModule} from '@angular/common';
-import {MyMaterialModule} from '../my-material/my-material.module';
-import {PlayerComponentComponent} from './player-component/player-component.component';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MyMaterialModule } from '../my-material/my-material.module';
+import { PlayerComponentComponent } from './player-component/player-component.component';
+import {MdSnackBarModule} from "@angular/material";
```

Comment:
Material Design modules must be imported through MyMaterialModule to keep up with the current standard.

Reply:
ups 😄